core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -83,14 +83,16 @@ class DataFileTestCase(LogBufferTestCase):
     particular validator class and collect the log records emitted.
     """
 
-    def validate(self, data_filename, validator_class, extra_meta_fields=None):
+    def validate(self, data_filename, validator_class, extra_meta_fields=None, relaxed_mode=None):
         """Validate a file with a Validator and return the log records."""
         meta_dict = {'data_filename': data_filename}
         if extra_meta_fields is not None:
             meta_dict.update(extra_meta_fields)
+        if relaxed_mode is None:
+            relaxed_mode = False
```

Comment:
In other words, this could be as follows:
```Python
def  validate([…], relaxed_mode=False)
```
and then the if-statement is not necessary.